### ISSUE TITLE
Tonnes of warnings when using HexColors through Carthage

### ISSUE THREAD

**Issue by fatuhoku** [mRs-/HexColors#37]
`$ carthage update --platform iOS` yields:

```
*** Fetching HexColors
...
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:30:14: warning: unused variable 'nilColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:37:14: warning: unused variable 'falseColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:44:14: warning: unused variable 'whiteColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:45:14: warning: unused variable 'whiteHexColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:52:14: warning: unused variable 'whiteColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:53:14: warning: unused variable 'whiteHexColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:60:14: warning: unused variable 'whiteColorAlpha' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:61:14: warning: unused variable 'whiteHexColorAlpha' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:68:14: warning: unused variable 'greyColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:69:14: warning: unused variable 'rgbColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:30:14: warning: unused variable 'nilColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:37:14: warning: unused variable 'falseColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:44:14: warning: unused variable 'whiteColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:45:14: warning: unused variable 'whiteHexColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:52:14: warning: unused variable 'whiteColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:53:14: warning: unused variable 'whiteHexColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:60:14: warning: unused variable 'whiteColorAlpha' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:61:14: warning: unused variable 'whiteHexColorAlpha' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:68:14: warning: unused variable 'greyColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:69:14: warning: unused variable 'rgbColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:30:14: warning: unused variable 'nilColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:37:14: warning: unused variable 'falseColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:44:14: warning: unused variable 'whiteColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:45:14: warning: unused variable 'whiteHexColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:52:14: warning: unused variable 'whiteColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:53:14: warning: unused variable 'whiteHexColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:60:14: warning: unused variable 'whiteColorAlpha' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:61:14: warning: unused variable 'whiteHexColorAlpha' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:68:14: warning: unused variable 'greyColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:69:14: warning: unused variable 'rgbColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:30:14: warning: unused variable 'nilColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:37:14: warning: unused variable 'falseColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:44:14: warning: unused variable 'whiteColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:45:14: warning: unused variable 'whiteHexColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:52:14: warning: unused variable 'whiteColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:53:14: warning: unused variable 'whiteHexColor' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:60:14: warning: unused variable 'whiteColorAlpha' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:61:14: warning: unused variable 'whiteHexColorAlpha' [-Wunused-variable]
project-dir/Carthage/Checkouts/HexColors/HexColorsTests/HexColorsTests.m:68:14: warning: unused variable 'greyColor' [-Wunused-variable]
```

**Comment by mRs-** [comment]
I don't use Carthage myself. Maybe you can hand in a merge request for this?

**Comment by mRs-** [comment]
Will be closed with Version 5.0
